config/mysql/mysql.go

```diff
@@ -62,13 +65,22 @@ func (m *Config) String() string {
 		m.Location = url.QueryEscape(DefaultLocation)
 	}
 
-	return fmt.Sprintf("%s:%s@tcp(%s:%d)/%s?loc=%s&parseTime=true",
+	addtl := m.AddtlDSNOptions
+	if m.ReadTimeout != "" {
+		addtl += fmt.Sprintf("&readTimeout=%s", m.ReadTimeout)
+	}
+	if m.WriteTimeout != "" {
+		addtl += fmt.Sprintf("&writeTimeout=%s", m.WriteTimeout)
+	}
+
+	return fmt.Sprintf("%s:%s@tcp(%s:%d)/%s?loc=%s&parseTime=true%s",
```

Comment:
With this setup, we're basically forcing all users to prefix the `AddtlDSNOptions` with `&`. Any thoughts on tacking that on for users as a convenience?